Makefile

```diff
@@ -257,6 +268,45 @@ helm-install-provider: helm-version check-openai-key
 		--timeout 5m       \
 		--kube-context kind-$(KIND_CLUSTER_NAME) \
 		--wait \
+		--set ui.service.type=LoadBalancer \
+		--set ui.image.registry=$(RETAGGED_DOCKER_REGISTRY) \
+		--set ui.image.tag=$(UI_IMAGE_TAG) \
+		--set controller.image.registry=$(RETAGGED_DOCKER_REGISTRY) \
+		--set controller.image.tag=$(CONTROLLER_IMAGE_TAG) \
+		$(if $(CI),--set controller.service.type=LoadBalancer) \
+		--set engine.image.registry=$(RETAGGED_DOCKER_REGISTRY) \
+		--set engine.image.tag=$(APP_IMAGE_TAG) \
+		--set providers.openAI.apiKey=$(OPENAI_API_KEY) \
+		--set providers.azureOpenAI.apiKey=$(AZUREOPENAI_API_KEY) \
+		--set providers.anthropic.apiKey=$(ANTHROPIC_API_KEY) \
+		--set providers.default=$(KAGENT_DEFAULT_MODEL_PROVIDER) \
+		$(if $(OPENAI_API_KEY),,--set querydoc.enabled=false) \
```

Comment:
I don't think we need to do these `ifs`. This is all for local anyway, plus the components "should" spin up regardless of whether or not there is a key